src/reviewRenderer/renderers/interactions/ExtendedTextInteraction.js

```diff
@@ -20,6 +20,8 @@
 /**
  * @author Ansul Sharma <ansultaotesting.com>
  */
+import $ from 'jquery';
+import _ from 'lodash';
```

Comment:
**suggestion:** maybe make [native alternatives](https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore) sens too